topics/assembly/tutorials/largegenome/tutorial.md

```diff
@@ -0,0 +1,587 @@
+  * reference_genome.fasta (e.g. of a closely-related species, if available). 
+* Outputs
+  * Busco table of genes found
+  * Quast HTML report, and link to Icarus contigs browser,  showing contigs aligned to a reference genome
+* Tools used: 
+  * Busco
+  * Quast
+* Input parameters: None required
+* Workflow steps:
+  * Polished assembly => Busco:
+  * First: predict genes in the assembly: using Metaeuk
+  * Second: compare the set of predicted genes to the set of expected genes in a particular lineage. Default setting for lineage: Eukaryota
+  * Polished assembly and a reference genome => Quast:
+  * For the tutorial we will use the Arabidopsis genome. This is not closely related to our Eucalyptus species but will give an idea of how to use Quast. 
+  * Contigs/scaffolds file: polished assembly
+  * Type of assembly: Genome
+  * Use a reference genome: Yes
+  * Reference genome: Arabidopsis genome
+  * Is the genome large (> 100Mbp)? Yes. (Our test data set won't be, but this will still give us some results). 
+  * All other settings as defaults, except second last setting: Distinguish contigs with more than 50% unaligned bases as a separate group of contigs?: change to No
+* Options:
+  * **Gene prediction:** 
+  * Change tool used by Busco to predict genes in the assembly: instead of Metaeuk, use Augustus (Metaeuk is meant to be faster; unsure which is better). 
+  * select: Use Augustus; Use another predefined species model; then choose from the drop down list. 
+ select from a database of trained species models. list here:  https://github.com/Gaius-Augustus/Augustus/tree/master/config/species
+  * Note: if using Augustus: it may fail if the input assembly is too small (e.g. a test-size data assembly). It can't do the training part properly. 
+  * **Compare genes found to other lineage:**
+  * Busco has databases of lineages and their expected genes. Option to change lineage. Not all lineages are available - there is a mix of broader and narrower lineages.
+  * list of lineages [here](https://busco.ezlab.org/list_of_lineages.html) 
+  * To see the groups in taxonomic hierarchies see [here](https://busco.ezlab.org/frames/euka.htm)
+  * For example,  if you have a plant species from Fabales, you could set that as the lineage. 
+  * The narrower the taxonomic group, the more total genes are expected. 
+{: .hands_on}
+
+*Assessment results*
+
+The output is a set of Quast and Busco reports. 
+
+Busco: As this is a test dataset, the assembly is small (~ 10 million base pairs, rather than ~ 1 billion base pairs), and likely missing most of the real genome (and genes). Thus, for this test case, we would expect most of the genes not to be found, but we can view the Busco results to see an example of how it works. Open the Busco short summary file: only 1 complete BUSCO has been found, out of 255 expected. You can re-run the workflow and change the lineage to "Embryophyta" to see that more BUSCOs are found (8 out of 1614). 
+
```

Comment:
I think it would be nice to include the BUSCO plots.

Reply:
added to list of enhancements